docs/modules/ROOT/pages/overview.adoc

```diff
@@ -1,14 +1,97 @@
 = Hazelcast Command-Line Client (CLC)
-:description: Short description with main points
+:url-github-clc: https://github.com/hazelcast/hazelcast-cloud-cli/blob/master/README.md 
+:description: You can use Hazelcast Command Line Client (CLC) to connect to and interact with clusters on Hazelcast Platform and Hazelcast Viridian, direct from the command line or through scripts.
 
-What does the product do?
-what are some examples of where you’d use it?
-Who is it for?
-What problem does the product solve?
-What modes are available and what are they?
-What (if any) limitations does the product have?
-How do you get started?
+{description}
+
+Hazelcast CLC simplifies the client setup process for Unix-like local environments. Within minutes of installation, you can start to perform common tasks on clusters.
```

Comment:
```suggestion
The Hazelcast CLC simplifies the client setup process. Within minutes of installation, you can start to perform common tasks on clusters.
```

It looks like we are adding an install script for Windows.

Reply:
Updated as requested.

Reply:
Agreed to leave pending merging of PR for Windows version.